### PR TITLE
Simplify add new exercise script

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This PR simplifies the add-new-exercise.ps1 script a bit. It is probably the most convenient to look at the individual commit messages, which should explain what the various simplifications do.

### CODE REVIEW

**Comment by davidelettieri** [comment]
Erik, thanks for your work!

I'm leaving for a vacation I cannot review the pr before the 5 of August. I hope this is not a problem. 

As a side note I was thinking if using https://github.com/ionide/Forge could help manipulate the fsproj.

**Comment by ErikSchierboom** [comment]
No problem at all. We still have @robkeim as another maintainer :) Enjoy your vacation!

File: add-new-exercise.ps1 [hunk 1]
```diff
@@ -1,67 +1,61 @@
...
+
+New-Item -ItemType File -Path "$exerciseDir/$projectName.fs" -Value "module $projectName"
+New-Item -ItemType File -Path "$exerciseDir/Example.fs" -Value "module $projectName"
+
+[xml]$proj = Get-Content $fsProj
 $proj.Project.ItemGroup[0].Compile[0].Include = "$projectName.fs"
 $proj.Project.ItemGroup[0].Compile[1].Include = "${projectName}Test.fs"
 $proj.Save($fsProj)
 
+./update-docs.ps1 -o $Exercise
```
Comment (ErikSchierboom) [inline add-new-exercise.ps1]: We now also generate the correct README file.

File: add-new-exercise.ps1 [hunk 2]
```diff
@@ -1,67 +1,61 @@
...
+
+$generatorsDir = Resolve-Path "generators"
+$generator = "type $projectName() =`n    inherit GeneratorExercise()`n"
+Add-Content -Path "$generatorsDir/Generators.fs" -Value $generator
+
+./generate-tests.ps1 $Exercise
```
Comment (ErikSchierboom) [inline add-new-exercise.ps1]: We now also add a stub test generator and generate the test suite.

File: generate-tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<#
+.SYNOPSIS
```
Comment (ErikSchierboom) [inline generate-tests.ps1]: I'll be adding this synopsis to the other ps1 files in a different PR.

**Comment by ErikSchierboom** [comment]
@robkeim This PR is now also ready to be reviewed

**Comment by robkeim** [comment]
Enjoy your vacation @davidelettieri :)

**Pull request merged**